packages/runtime/container-runtime/src/pendingStateManager.ts

```diff
@@ -183,7 +181,7 @@ export class PendingStateManager implements IDisposable {
 		const pendingMessage: IPendingMessage = {
 			type: "message",
 			messageType: type,
-			clientSequenceNumber,
+			clientSequenceNumber: -1, // dummy value (not to be used anywhere)
```

Comment:
Do we need this due to serialization format and back compat? Or something else?

Reply:
It's for back-compat purposes. This interface has been deprecated (for public export) by https://github.com/microsoft/FluidFramework/pull/13917, so this property can also be removed when it's no longer publicly exported.